packages/ui/src/Box.tsx

```diff
@@ -90,6 +90,7 @@ export class Box extends React.Component<BoxProps, {}> {
     paddingX: (value) => ({paddingLeft: value * SPACING, paddingRight: value * SPACING}),
     paddingY: (value) => ({paddingTop: value * SPACING, paddingBottom: value * SPACING}),
     padding: (value) => ({padding: value * SPACING}),
+    zIndex: (value) => ({zIndex: value}),
```

Comment:
Just to follow the pattern of other params, I'd update this to `zIndex: (value) => ({zIndex: value ? value : "auto"}),`